witchcraft-server/src/logging/format.rs

```diff
@@ -67,6 +69,18 @@ impl LogFormat for TraceLogV1 {
     type Reporter = StandardReporter<Self>;
 }
 
+impl LogFormat for AuditLogV3 {
+    const TYPE: &'static str = "audit.3";
+
+    const FILE_STEM: &'static str = "audit.3";
```

Comment:
Should this just be `"audit"`?

Reply:
That's the stem for the old v2 audit logs - we switched file names for v3 for whatever reason. I'll PM you a link to the internal java impl